src/main/java/com/openshift/jenkins/plugins/pipeline/OcWatch.java

```diff
@@ -153,16 +151,16 @@ public Void run() {
                                 listener.getLogger().println( "Checking watch output again in " + reCheckSleep + "ms" );
                                 Thread.sleep( reCheckSleep );
 
-                            } while ( (exitStatus = dtc.exitStatus(filePath,launcher)) == null );
+                            } while ( (exitStatus = exitStatusRaceConditionBugWorkaround(dtc, filePath,launcher)) == null );
```

Comment:
this could loop forever if something goes wrong and we can't get the exit code (existStatusRace... returns null after 30 retries), right?  I think we want a backstop.